lib/tether/tether_linux.go

```diff
@@ -75,18 +76,17 @@ func (t *tether) childReaper() error {
 				// general resiliency
 				defer func() {
 					if r := recover(); r != nil {
-						fmt.Printf("Recovered in childReaper %v", r)
+						fmt.Printf("Recovered in childReaper %s", debug.Stack())
```

Comment:
This should be really rare, in fact it shouldn't be happening at all so when it happens I rather want to see everything